includes/Aws/CloudFront/Invalidation.php

```diff
@@ -34,18 +34,12 @@ public function invalidate(array $items) {
                 ],
             ]);
             return true;
-        } catch (AwsException $error) {
-            $message = $error->getAwsRequestId() . PHP_EOL;
-            $message .= $error->getAwsErrorType() . PHP_EOL;
-            $message .= $error->getAwsErrorCode() . PHP_EOL;
-        } catch (CloudFrontException | Exception $error) {
-            $message = $error->getMessage();
+        } catch (AwsException | CloudFrontException | Exception $error) {
+            $response = new WP_Error('cloudfront_invalidation_error', sprintf( __("Could not invalidate CloudFront distribution: %s", 'simply_static_deploy'), $error->getMessage()), [
+                'status' => 400,
+            ]);
+            do_action('simply_static_deploy_error', $response);
+            return $response;
```

Comment:
By including `Exception` you might as well remove the other two? 
Note that this is an OR statement, so you're catching `AwsException` OR `CloudFrontException` OR `Exception`, but since every exception is a subclass of `Exception`, this will always trigger. 
Unless you mean in this case an `Exception` specific to the namespace that you're in. In any case, it confuses me a little. 🙂

Reply:
That's what I said in my last sentence:

> Maybe the type hints are a bit explicit now, but at least you know what you're getting.

They're there because they were there. It's a nice hint of what you 'could expect', but not needed. But if it confuses you, I'll remove it.

Reply:
They're gone 👋